fastapi/exceptions.py

```diff
@@ -13,8 +16,10 @@ def __init__(
 
 
 class RequestValidationError(ValidationError):
-    pass
+    def __init__(self, errors: Sequence[ErrorList]) -> None:
+        super().__init__(errors, Request)
```

Comment:
`pydantic` now requires ValidationError to receive a type to indicate as the type that failed validation. The value is *only* used as a way to obtain a name to put in the exception traceback. Given that `RequestValidationError` is collecting all of the errors from the various dependencies that may succeed or fail, I think just calling it out as the Request is an easy way to pick a type to use here, and it is still actually pretty useful for debugging because it will indicate the problem occurred before your endpoint-specific code had even started.

(This will also make it easier to diagnose problems in submitted issues/gitter/etc., especially if the submitter is light on details.)